resources/views/modals/submit_patrol_log_modal.blade.php

```diff
@@ -65,13 +65,19 @@
               </div>
               <label id="patrol-end_time-error" class="error mt-2 text-danger" for="patrol-end-time" hidden></label>
             </div>
-
             <div class="form-group">
               <label>Patrol Description (required)</label>
               <textarea class="form-control" id="patrol-details" rows="6"></textarea>
               <label id="patrol-details-error" class="error mt-2 text-danger" for="patrol-details" hidden></label>
             </div>
-
+              <div class="form-group">
+                  <label>Select AOP</label>
+                  <select class="js-example-basic-multiple" multiple="multiple" id="patrol-aop" style="width:100%">
+                      <option value="{{$constants['aops']['BC']}}">Blaine County</option>
```

Comment:
This isn't that future proof in case we add more AOPs to constants,
Try using this as an example: 
![](https://i.imgur.com/QWSDg5i.png)

You can find it in add_member_action_modal.blade.php line 59.